src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import Button from "./../Button/Button";
+import Icon from "./../Icons/Icon";
+import { Box, useMultiStyleConfig } from "@chakra-ui/react";
+import { SelectedItems } from "./MultiSelectTypes";
+
+export interface MultiSelectMenuButtonProps {
+  /** The id of the multiSelect using this button. */
+  multiSelectId: string;
+  /** The label of the multiSelect using this button. */
+  label: string;
+  /** The open status of the multiselect menu. */
+  isOpen: boolean;
+  /** The selected items state (items that were checked by user). */
+  selectedItems: SelectedItems;
+  /** The callback function for the menu toggle. */
+  onMenuToggle?: () => void;
+  /** The action to perform for clear/reset button of multiselect. */
```

Comment:
Make sure to update `multiselect` to `MultiSelect`.

Reply:
fixed

Reply:
fixed